index.html

```diff
@@ -1122,11 +1129,290 @@ <h5>
+        </p>
+        <p>
+          Even users unwilling to share their information in such a scheme
+          could be affected by prompt fatigue and potentially risk exclusion
+          from using these services.
+        </p>
+        <h5>
+          Mitigating unnecessary requests for non-government credentials
+        </h5>
+        <p>
+          For non-government-issued credentials, it is recommended that the
+          user agent understand the requested credential format and its privacy
+          attributes, and build a risk framework that informs the context that
+          is shown to the user, as well as the amount of friction that is
+          appropriate for each credential type. Protocols and formats involved
+          in the exchange of these credentials are generally expected to
+          support features such as selective disclosure and unlinkability, but
+          these features might not always be appropriate or necessary in the
+          exchange of information, especially when it concerns low-risk
+          credentials such as cinema tickets.
+        </p>
+        <p>
+          A user agent that recognizes the type of credential being requested
+          is encouraged to customize its permission experience to best suit the
+          requested credential and help users understand the consequences of
+          sharing it.
+        </p>
+        <p>
+          User agents can not be expected to understand all credential
+          requests. A user agent that does not recognize the type of credential
+          being requested is advised to significantly increase user friction in
+          their permission experience, and very clearly communicate the risks
+          of sharing unknown credentials with websites to the user. Note that
+          this could require integration between <a href=
+          "#multiple-user-agents">different user agents</a> to apply
+          appropriate levels of friction and transparency. For example, a
+          browser might delegate knowledge about credential requests to the
+          operating system, which might require wallets to register known
+          credential types and reject an exchange request for an unknown
+          credential type.
+        </p>
+        <p class="issue" data-number="100">
+          The need to provide users with appropriate transparency conflicts
+          with the desire to enable the ecosystem to develop new credential
+          formats without explicit user agent buy-in.
+        </p>
+        <p class="issue" data-number="117">
+          Is it a good idea for the group to maintain a registry of request
+          types and/or <a href=
+          "https://github.com/w3c-fedid/digital-credentials/issues/85">credential
+          types</a>?
+        </p>
+        <h5>
+          Reporting abuse
+        </h5>
+        <p class="issue" data-number="267">
+          Consider an interoperable abuse reporting system for verifiers making
```

Comment:
Thank you for including this as a potential mitigation technique. While it's a new idea, if we find it works appropriately, I think it could also be applied more generally to other permission policy abuses too which is encouraging. One step at a time though.

Reply:
Thanks! It's @npdoty's idea though. We should definitely explore it further.